frontends/blif/blifparse.cc

```diff
@@ -281,12 +300,16 @@ void parse_blif(RTLIL::Design *design, std::istream &f, std::string dff_name, bo
 						const_v.bits[i] = v[n-i-1] != '0' ? State::S1 : State::S0;
 				}
 				if (!strcmp(cmd, ".attr")) {
-					if (obj_attributes == nullptr)
+					if (obj_attributes == nullptr) {
+						log_error("No object to attach .attr too.\n");
 						goto error;
```

Comment:
This doesn't make a lot of sense. `log_error` does not return, so the `goto` is never reached and the user is never told in which line the error occurred. (same issue with parameters below)

Reply:
Indeed, switched to log_warning prior to jumping to the common error message.